roles/rhsm/tasks/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 
 - name: "Install Satellite certificate (if applicable)"
-  command: "rpm -Uh --force http://{{ rhsm_server_hostname }}/pub/katello-ca-consumer-latest.noarch.rpm"
+  yum:
```

Comment:
If we decide to keep this, we should probably change to the `package` module. I believe that's still best practice.

Reply:
I don't think so @etsauer, as this will call the `yum` underneath. As we are not targeting any other OS rather than RHEL or RHEL based, using `package` manager will add an extra task that is not required.

https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/packaging/os/package.py#L46